src/FluentValidation.AspNetCore/CustomizeValidatorAttribute.cs

```diff
@@ -1,4 +1,4 @@
-﻿#region License
+#region License

```

Comment:
I think this was the BOM... I was using Github Codespaces to do the work and couldn't find a way to restore it.  Let me know if this is bothersome and I'll pull it down on my machine and fix it.

Reply:
That's fine, some of the older files in the repo (like this) have some minor formatting issues from before I added an `.editorconfig` file. GH is just correcting this :)